### PR TITLE
Add issue templates

### LINKED ISSUE #555: Clicking the green "New issue" button doesn't start a new issue
See the attached screen shot.  Is this intentional?    I at first didn't notice the "Open a blank issue" link so the page that comes up after clicking "New issue" is a little confusing.

Screen Shot 2021-09-02 at 9 46 40 AM

### PR DESCRIPTION
**Pull request by awvwgk**
You can checkout the available templates here:
https://github.com/awvwgk/fortran-package-manager/issues/new/choose

Provides:
- Issue form for bug reports
- Issue form for packaging issues
- Issue form for feature requests
- Issue form for specification proposals
- Issue template for blank issue
- Link to fpm discussion board
- Link to Fortran-lang discourse
- Link to Fortran-lang mailing list

Closes #555 

Screenshot:

<details><summary>New issue</summary>

image
</details>

<details><summary>Bug report form</summary>

image
</details>
<details><summary>Packaging issue form</summary>

image
</details>
<details><summary>Feature request form</summary>

image
</details>
<details><summary>Specification proposal form</summary>

image
</details>
<details><summary>Blank issue</summary>

image
</details>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/02_packaging.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Packaging Issue
+description: Porting or packaging a project to fpm `
```
Comment (brocolis) [inline .github/ISSUE_TEMPLATE/02_packaging.yaml:2]:
```suggestion
description: Porting or packaging a project to fpm
```

**Review by brocolis — APPROVED** [review]

**Review by brocolis — APPROVED** [review]

**Comment by awvwgk** [comment]
@rouson @milancurcic Does this change look okay to you?

**Review by milancurcic — APPROVED** [review]
I like it, thanks!

**Comment by awvwgk** [comment]
With two approvals I'll go ahead and merge later today.

**Pull request merged**